src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -160,7 +160,9 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
             <Typography sx={{ fontWeight: 'bold' }}>WBS #: </Typography>
           </Grid>
           <Grid item xs={10}>
-            <MUILink href={`${routes.PROJECTS}/${wbsPipe(changeRequest.wbsNum)}`}>{wbsPipe(changeRequest.wbsNum)}</MUILink>
+            <MUILink component={Link} to={`${routes.PROJECTS}/${wbsPipe(changeRequest.wbsNum)}`}>
```

Comment:
can you rename these while you're here so it's like the other ones: <Link component={RouterLink}